api/models.py

```diff
@@ -172,7 +172,9 @@ class Contract(models.Model):
     modified_by = models.ForeignKey(
         to=User, related_name="+", on_delete=models.CASCADE
     )  # No backwards relation to these Fields
-
+    
+    def __str__(self):
+        return self.name
```

Comment:
I reconcidered: `user.last_name` - `self.name` would be better
Reason:  A lot of ppl choose the same/similar names for their contracts